sympy/physics/mechanics/joint.py

```diff
@@ -543,3 +543,137 @@ def _set_linear_velocity(self):
+    """Sliding (Prismatic) Joint.
+
+    Explanation
+    ===========
+
+    It is defined such that the child body translates with respect to the parent
+    body along the body fixed parent axis. The point of joint (sliding joint) is defined
+    by two points in each body which coincides initially.
+
+    Parameters
+    ==========
+
+    name : string
+        The Joint's name which makes it unique.
+    parent : Body
+        The parent body of joint.
+    child : Body
+        The child body of joint.
+    coordinates: dynamicsymbol, optional
+        Coordinates of joint.
+    speeds : dynamicsymbol, optional
+        Speeds of joint.
+    parent_joint_pos : Vector, optional
+        Defines the joint's point where the parent will be connected to child.
+        Default value is masscenter of Parent Body.
+    child_joint_pos : Vector, optional
+        Defines the joint's point where the child will be connected to parent.
+        Default value is masscenter of Child Body.
+    parent_axis : Vector, optional
+        Axis of parent frame which would be be aligned with child's
+        axis. Default is x axis in parent's frame.
+    child_axis : Vector, optional
+        Axis of child frame which would be aligned with parent's
+        axis. Default is x axis in child's frame.
+
+    Attributes
+    ==========
+
+    name : string
+        The joint's name.
+    parent : Body
+        The joint's parent body.
+    child : Body
+        The joint's child body.
+    coordinate : dynamicsymbol, optional
+        Coordinate of the joint.
+    speed : dynamicsymbol, optional
+        Speed of the joint.
+    parent_joint : Point
+        The joint's point where parent body is connected to the joint.
+    child_joint : Point
+        The joint's point where child body is connected to the joint.
+    parent_axis : Vector
+        The axis of parent frame.
+    child_axis : Vector
+        The axis of child frame.
+
+    Examples
+    =========
+
+    This is an example of linear mass spring damper where we will do all kinematics
+    using SlidingJoints.
+
+    >>> from sympy import symbols
+    >>> from sympy.physics.mechanics import SlidingJoint, Body
+    >>> from sympy.physics.vector import dynamicsymbols, ReferenceFrame
+
+    Declaring the mass, frame, speeds, coordinates etc isn't necessary as `Body` and
+    `SlidingJoint` can declare it themselves. But we would be declaring them for better understanding.
+
+    >>> x, v = dynamicsymbols('x v')
+    >>> m = symbols('m')
+    >>> N = ReferenceFrame('N')
+
+    Declaring the bodies.
+
+    >>> C = Body('C', frame=N)
+    >>> P = Body('P', frame=N, mass=m)
+
+    Bodies are connected through their masscenters.
+
+    >>> J = SlidingJoint('J', C, P, coordinates=x, speeds=v)
+
+    We can check the kinematics now.
+
+    >>> P.masscenter.pos_from(C.masscenter)
+    x(t)*N.x
+    >>> P.masscenter.vel(N)
+    v(t)*N.x
+    >>> J.kdes()
+    [v(t) - Derivative(x(t), t)]
```

Comment:
Similar to pin joint you could also add a basic chained slider example. There is this one in PyDy: https://github.com/pydy/pydy/blob/master/pydy/models.py#L16, just multiple masses attached together by springs.

Reply:
I already have `linear mass spring damper` example in docstrings, isn't one enough in docstring? Others can go into examples module?

Reply:
I don't see it here. Where is it?

In PinJoint we have two examples in the docstring: 1) shows the simplest pin joint and it's class features, 2) super simple multibody system kinematics (simple double pendulum). If we follow that pattern it makes sense to have a 3 body slider here.

Reply:
Please check the examples now.